src/components/SQForm/SQFormDatePicker.js

```diff
@@ -58,6 +58,9 @@ function SQFormDatePicker({
         shouldDisableDate={setDisabledDate}
         value={value}
         onChange={handleChange}
+        onClose={() => {
+          helpers.setTouched();
```

Comment:
Hm. I'm conflicted on this. If we open the window and don't change anything is the form touched?
Would it make more sense to just run the validation via `validateForm` from FormikContext?

Reply:
I did try that, but I don't think it'll validate an untouched field (which this will be) - but I also just might not know what I'm doing.

Perhaps there's a smart way to see if value === originalValue, and if different, mark touched?

Reply:
Isn't this sort of how the textfield works? If you tab out of a required textfield without changing anything it's also considered touched, right?

Reply:
Is it? I'm not entirely sure. If literally touching a field without changing the value makes a field touched by Formik's standards then I'm totally okay with solution to this issue.

Reply:
I'm not super familiar with Formik, but that seems to be how it works in the SQForm validation story

Reply:
I actually just completed the Formik tutorial and yes, if you tab out of a field without changing the value it's still considered touched.

Reply:
> I actually just completed the Formik tutorial and yes, if you tab out of a field without changing the value it's still considered touched.

Great! :shipit:

Reply:
A Formik tutorial? Cool!
Touched or dirty is HTML jargon. A field is not considered dirty until a user interacts with the field. Tabbing or touching a field counts as an interaction.